packages/ui-kit-components/src/components/Branding/Logo/Logo.tsx

```diff
@@ -1,19 +1,24 @@
-import { useTheme } from '@mui/material/styles';
+import { useTheme } from '@mui/material';
 
-import { LogoLarge, LogoMaximized, LogoSmall } from '../raw-icons';
+import {
+  darkLogoColor,
+  lightLogoColor,
+  LogoLarge,
+  LogoMaximized,
+  LogoSmall,
+} from '../branding';
 
 interface Props {
   variant?: string;
 }
 
 const Logo: React.FC<Props> = (svgProps) => {
   const theme = useTheme();
-  const color = theme.palette.logo;
+  const color = theme.palette.mode === 'dark' ? lightLogoColor : darkLogoColor;
 
   switch (svgProps.variant) {
     case 'maximized':
       return <LogoMaximized sx={{ color: color, height: 53, width: 154 }} />;
```

Comment:
Does the height and width are with Unit `px`, it is better to specify the units.

Reply:
I'd actually say it is relatively common for number values to be provided for styles in JSX, and it is assumed that without units the numbers refer to PX.